doc/man3/CTLOG_STORE_new.pod

```diff
@@ -65,20 +71,23 @@ invalid if it is missing a "key" or "description" field.
 Both B<CTLOG_STORE_load_default_file> and B<CTLOG_STORE_load_file> return 1 if
 all CT logs in the file are successfully parsed and loaded, 0 otherwise.
 
+B<CTLOG_STORE_add0_log> returns 1 if the log was added successfully, 0 otherwise.
+
 =head1 SEE ALSO
 
 L<ct(7)>,
 L<CTLOG_STORE_get0_log_by_id(3)>,
+L<CTLOG_new(3)>,
 L<SSL_CTX_set_ctlog_list_file(3)>
 
 =head1 HISTORY
 
-CTLOG_STORE_new_ex was added in OpenSSL 3.0. All other functions were
-added in OpenSSL 1.1.0.
+CTLOG_STORE_new_ex and CTLOG_STORE_add0_log were added in OpenSSL 3.0.
```

Comment:
This will need an update since it's too late for 3.0.

Reply:
I guess this is only relevant once the "otc review" happened, whenever that is?